scripts/feature_toggle_report.py

```diff
@@ -93,23 +90,39 @@ def main(annotations_dir, toggle_data_dir, output_file_path, show_state, env, to
         if os.path.isdir(os.path.join(toggle_data_dir, path)) and env_name_search:
             env_data_paths.append((os.path.join(toggle_data_dir, path), env_name_search.group('env')))
 
-    total_info = {}
+    # if no env folders were found at toggle_data_dir, assume that dir contains data for one env
+    if not env_data_paths:
+        env_name_search = env_name_pattern.search(os.path.basename(toggle_data_dir))
+        if env_name_search:
+            env_data_paths.append((toggle_data_dir, env_name_search.group('env')))
+        else:
+            raise Exception('Directory at {} does not match required structure, see readme for more info'.format(toggle_data_dir))
+
+
+    idas = {}
     for env_data_path, env_name in env_data_paths:
         # if an env is specified in requested_envs, filter out everyother env
         # if no env is specified, assume all envs are valid
         if requested_envs and env_name not in requested_envs:
             LOGGER.debug("Skip reading toggle state data for {} env".format(env_name))
             continue
-        total_info[env_name] = {}
 
         # add data for each ida
-        if show_state:
-            add_toggle_state_to_idas(total_info[env_name], env_data_path, configuration.get("ida", defaultdict(dict)))
+        add_toggle_state_to_idas(idas, env_data_path, configuration.get("ida", defaultdict(dict)), env_name=env_name)
+
+        add_toggle_annotations_to_idas(idas, annotations_dir, configuration.get("ida", defaultdict(dict)))
+
+    toggle_data = []
+    if verbose_report:
+        for ida in idas.values():
+            toggle_data.extend(ida.get_full_report())
+    else:
+        for ida in idas.values():
+            toggle_data.extend(ida.get_toggles_data_summary())
 
-        add_toggle_annotations_to_idas(total_info[env_name], annotations_dir, configuration.get("ida", defaultdict(dict)))
 
     renderer = CsvRenderer()
-    renderer.render_csv_report(total_info, output_file_path, toggle_type_filter, ["name"])
+    renderer.render_csv_report(toggle_data, output_file_path, toggle_type_filter, ["name", "ida_name", "code_owner", "oldest_created", "newest_modified"])
```

Comment:
Nit: Consider creating a well-named variable for `["name", ...]` on the line before.